### PR TITLE
Feature/XIVY-12991 notification

### PR DESCRIPTION
**Pull request by weissreto**

### CODE REVIEW

**Review by ivy-lgi — APPROVED** [review]

File: testProjects/latest/Performance/processes/Notification/Setup.p.json [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+        "labelOffset" : { "x" : 13, "y" : 33 }
+      }
+    }, {
+      "id" : "f14",
+      "type" : "Script",
+      "name" : "Setup Mail Server",
+      "config" : {
+        "output" : {
+          "code" : [
+            "import ch.ivyteam.ivy.configuration.restricted.IConfiguration;",
```
Comment (alexsuter) [inline testProjects/latest/Performance/processes/Notification/Setup.p.json:140]: define in ivy.yaml?

File: testProjects/latest/Performance/processes/Notification/Setup.p.json [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+        { "id" : "f4", "to" : "f5", "color" : "default" }
+      ]
+    }, {
+      "id" : "f5",
+      "type" : "Script",
+      "name" : "Turn on notification",
+      "config" : {
+        "output" : {
+          "code" : [
+            "import ch.ivyteam.ivy.notification.channel.NotificationChannel;",
```
Comment (alexsuter) [inline testProjects/latest/Performance/processes/Notification/Setup.p.json:56]: define in ivy.yaml?

File: testProjects/latest/Performance/processes/Notification/Setup.p.json [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+        { "id" : "f9", "to" : "f8" }
+      ]
+    }, {
+      "id" : "f8",
+      "type" : "Script",
+      "name" : "Delete users",
+      "config" : {
+        "output" : {
+          "code" : [
+            "import ch.ivyteam.ivy.security.IUser;",
```
Comment (alexsuter) [inline testProjects/latest/Performance/processes/Notification/Setup.p.json:90]: needed?

File: Jenkinsfile [hunk 4]
```diff
@@ -86,10 +86,11 @@ def prepareIvyContainer(String version) {
 }
 
 def runPerformanceTests(String version) {
-  container = docker.image("ivy-$version:${env.BUILD_ID}").run()
+  mailContainer = docker.image("greenmail/standalone").run();
+  ivyContainer = docker.image("ivy-$version:${env.BUILD_ID}").run(" --link ${mailContainer.id}:mail");
```
Comment (alexsuter) [inline Jenkinsfile:90]: if ivyContainer fails with an exception then I think we don't run through finally to destroy mailContainer?

File: Jenkinsfile [hunk 5]
```diff
@@ -107,15 +108,22 @@ def runPerformanceTests(String version) {
       runPerformanceTest(version, "businessDataSearch", "performance/pro/Performance/1848AB6ADCA8F207/searchBusinessData.ivp")
       runPerformanceTest(version, "businessDataFind", "performance/pro/Performance/1848AB6ADCA8F207/findBusinessData.ivp")
       runPerformanceTest(version, "task", "performance/pro/Performance/184D1FD2D64C0ADB/task.ivp")
+      if (version.equals("dev")) {
```
Comment (alexsuter) [inline Jenkinsfile:111]: rather mainting this version? maybe ask if notification feature is available?

**Review by alexsuter — APPROVED** [review]

**Pull request merged**